src/app/static/src/app/components/downloadResults/UploadModal.vue

```diff
@@ -166,7 +193,9 @@
                 if (this.uploadableFiles) {
                     return Object.keys(this.uploadableFiles).reduce((sections, key) => {
                         if (outputFileTypes.includes(key)) {
-                            sections[0][key] = this.uploadableFiles[key];
+                            if (Object.keys(this.getAvailableOutputFiles(key)).length > 0) {
```

Comment:
This seems like quite a confusing way of expressing this - the name `getAvailableOutputFiles` implies it might return more than one, which never happens, and it's not idea to have to call it twice. How about replacing `getAvailableOutputFiles` with `outputFileIsAvailable` which just returns a bool, then use `this.uploadableFiles[key]` as before?